travis-ymls/ipywidgets.travis.yml

```diff
@@ -20,13 +20,6 @@ python:
     - 3.7
     - 3.8
     - 3.9
-addons:
-  apt_packages:
-    - pandoc
-  firefox: latest
-services:
-  - xvfb
```

Comment:
You can manually install these using "addons/apt" and start the service using "systemctl start" are you not able to run yarn test with that ?